src/Compilers/CSharp/Portable/Symbols/Attributes/PEAttributeData.cs

```diff
@@ -102,7 +102,10 @@ private void EnsureAttributeArgumentsAreLoaded()
                 TypedConstant[]? lazyConstructorArguments = null;
                 KeyValuePair<string, TypedConstant>[]? lazyNamedArguments = null;
 
-                if (!_decoder.GetCustomAttribute(_handle, out lazyConstructorArguments, out lazyNamedArguments))
+                // If an attribute argument contains a generic parameter, the decoder needs to know the attribute type.
```

Comment:
>If an attribute argument contains a generic parameter, the decoder needs to know the attribute type. [](http://example.com/codeflow?start=19&length=100)

I think, I am missing something here. Why would this be the case? The attribute type in an attribute isn't providing a generic context, it doesn't define new type parameters, which could be referenced in arguments

Reply:
I think I have explained this above - the attribute type can be for example `A<B<int>.E>` which has the generic argument `B<int>.E` for its generic parameter `T` which can be referenced in the attribute constructor's argument as e.g. `T arg`.